build.gradle

```diff
@@ -50,18 +50,24 @@ task integrationTest(type: Test) {
 }
 
 tasks.withType(Test) {
-    environment 'GRADLE_RELEASE_REPOSITORY','https://services.gradle.org/distributions'
+    environment 'GRADLE_RELEASE_REPOSITORY', 'https://services.gradle.org/distributions'
 }
 
 dependencies {
+    compileOnly 'org.projectlombok:lombok:1.18.24'
+    annotationProcessor 'org.projectlombok:lombok:1.18.24'
+
+    testCompileOnly 'org.projectlombok:lombok:1.18.24'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.24'
+
```

Comment:
can you please extract the version to a property so that changing gets easier in the future? We currently have those in `gradle.properties`.